packages/react/src/ActionMenu/ActionMenu.tsx

```diff
@@ -476,8 +476,8 @@ const ActionMenuItem = ({
         className,
       )}
       role={roleTypeMap[type || 'single']}
-      aria-checked={as === 'a' || type === 'none' ? undefined : selected ? 'true' : 'false'}
-      aria-disabled={disabled ? 'true' : 'false'}
+      aria-checked={as === 'a' || type === 'none' ? undefined : selected}
+      aria-disabled={disabled}
```

Comment:
Was this refactoring relevant to resolving the aria-checked / aria-expanded issue? It seems to be minor and potentially noop, but can we still capture these in changesets. You can call it out as general internal refactoring to other component props like `aria-disabled`. Helps us all keep an eye out for regressions and prevent indirection because current changeset implies the update only affects aria-checked and aria-expanded.

Reply:
Just minor tidying to, imo, help with readability.

Updated the changeset 👍